paymentsheet/res/values-b+es+419/strings.xml

```diff
@@ -17,6 +17,10 @@
   <string name="stripe_failure_connection_error">Estamos teniendo problemas para conectar con nuestro proveedor de servicios de pagos. Comprueba si tienes conexión a Internet y vuelve a intentarlo.</string>
   <!-- When an internal error is reported when a google pay payment is started. -->
   <string name="stripe_google_pay_error_internal">Se produjo un error interno.</string>
+  <!-- Text providing link to terms for ACH payments -->
+  <string name="stripe_paymentsheet_ach_continue_mandate"><![CDATA[Al continuar, aceptas autorizar pagos conforme a <terms>estas condiciones</terms>.]]></string>
+  <!-- Mandate text with link to terms when saving a bank account payment method to a merchant (merchant name replaces %@). -->
+  <string name="stripe_paymentsheet_ach_save_mandate"><![CDATA[Al guardar tu cuenta bancaria para %@, aceptas autorizar pagos conforme a estas <terms>condiciones</terms>.]]></string>
```

Comment:
Why did we get %@ instead of the Android format? Did iOS change their script and is uploading in a different format? If so we need to fix that

Reply:
The translations currently use %@ instead of the universal %s format. The fix would be to update the translations through lokalise, just not sure how long it will take